release/trigger-branch

```diff
@@ -1,2 +1,2 @@
 VERSION=1.17
-STEP=1
\ No newline at end of file
+STEP=1
```

Comment:
You likely don't want to actually cause a trigger, although in reality there is no automation to cause the branch to actually run. I'd just remove this change.

Reply:
In my testing though, I did update the VERSION to 1.18 so that the appropriate branches were used in the steps.